### ISSUE TITLE
Backend: Tabs and tabpanel -> wrong aria, and missing tabindex

### ISSUE THREAD

**Issue by angieradtke** [joomla/joomla-cms#44831]
Hi 

had contact to a screenreader-user

Issue:

Backend: Tabs and Tabpanel   can be operated using the arrow buttons (OK), but the tabs are also in the focus order (tab order). Only the selected tab may be in the focus order. When I tab through, each tab is "selected", but the selected tab is also "expanded". 
We are using the the wrong aria-attributes for the tab-buttons and the tabindex for the button is is not set.

tab-button should  have : 'aria-selected instead of aria-expanded (for accordion-button correct) and we need to set the tabindex.

`     tabObj.accordionButton.removeAttribute('aria-disabled');
      tabObj.tabButton.setAttribute('aria-selected', false);
      tabObj.tabButton.setAttribute('tabindex', -1) ;`

This  we can change in \media\vendor\joomla-custom-elements\joomla-tab.js

``

**Comment by richard67** [comment]
> Fixed Js-Files as text-file attached, because I can not find this file in the build -folder to make pull request .-(

@angieradtke Well that's because joomla-custom-elements is an NPM dependency. The JS would need to be fixed upstream, and when merged there it would need to make a new release of the joomla-custom-elements and then make a PR in this repo here to update that dependency.

The upstream repository should be this one: https://github.com/joomla-projects/custom-elements

With which version did you observe the issue? 5.2.3? Or 5.3-dev?

**Comment by angieradtke** [comment]
> ? 5.2.3

Yes  5.2.3

**Comment by angieradtke** [comment]
I have tried the pull request again, please check if this is correct? Thank you very much .-)
https://github.com/joomla-projects/custom-elements/pull/242

**Comment by chmst** [comment]
I have tested this item successfully. It cannot be tested as usual. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44831">issues.joomla.org/tracker/joomla-cms/44831</a>.</sub>

**Comment by chmst** [comment]
https://github.com/joomla/joomla-cms/issues/44831#issuecomment-2640328810 is merged.

**Comment by richard67** [comment]
Well I think it would need a new release or tag upstream before we can fetch that into the CMS with a new PR.

**Comment by richard67** [comment]
New release of custom elements is available, I will create a PR to get it into the CMS here. Just have to clarify if that has to go into 5.3-dev or 5.2-dev.

**Comment by richard67** [comment]
Closing as having a pull request. Please test #44888 . Thanks in advance.

@angieradtke I know it would mean that you test your own code, but I think a test by you would be still good so you can verify that the fix will be correctly included in the CMS.

@chmst Would be cool if you could test, too.

**Comment by richard67** [comment]
Hmm, still no tests for PR #44888 .
